### ISSUE TITLE
Upload build artefacts

### ISSUE THREAD

**Issue by probonopd** [rodlie/draco#39]
Currently Travis CI builds every git push and every PR. Wouldn't it be nice if it would create debs (since it is running on Ubuntu anyway) and upload them to GitHub Releases for easier testing?

Let me know if you would like to have this, then I will look into it.

**Comment by rodlie** [comment]
I use OBS, maybe a better solution?

* https://build.opensuse.org/package/show/home:rodlie:draco-master/draco
* https://build.opensuse.org/package/show/home:rodlie:draco-release/draco

**Comment by probonopd** [comment]
Yes, clearly much better! At least for releases. (For PRs and such Travis may still have value.) Would you know how to add a deb for xenial and bionic there?

**Comment by rodlie** [comment]
I know how to make deb's from scratch, but not how it works on OBS or through misc tools.

**Comment by probonopd** [comment]
https://github.com/probonopd/draco/releases/tag/continuous - "something" got built, testing it now

**Comment by probonopd** [comment]
The "something" is working for me on a Xubuntu Bionic Live iSO. In case you would like to test: Run Live ISO, download deb from https://github.com/probonopd/draco/releases/tag/continuous, `sudo apt install -y ./draco_0.9.1_amd64.deb` (it installs a lot of additional packages including some Gnome-related stuff - why?), log out, log into Draco session. Works for me.

Would you like a PR for it?

**Comment by rodlie** [comment]
Will test later today, don't have access to my main computer at the moment.

I did test Xubuntu Bionic Live earlier today, and was not able to get lightdm to start draco through regular means.

**Comment by rodlie** [comment]
btw, the gnome stuff is probably related to xscreensaver-demo and pnmixer, as they both uses gtk2 (they do however not require anything related to gnome, this is a packaging issue on Ubuntu IMHO).

**Comment by probonopd** [comment]
Possibly I was mixing up Gnome and Gtk.

**Comment by rodlie** [comment]
There are alternatives if you don't want gtk stuff. pnmixer should be easy to replace (and is just a recommended app), there are probably several qt alternatives available. pnmixer is good enough for my usage (I use the key bindings and sometimes scroll over the tray to adjust the volume). xscreensaver is currently a hard requirement (but not xscreensaver-demo). I'm willing to add support for more screensaver apps if needed (they must/should be able to control the same way as xscreensaver).

Audio control is always something that can be included, there should be enough examples available (and/or direct code to take, must be (L)GPL2 compatible).
